gateway/utils/tmpl/go.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 // GoTemplate executes a go template
-func GoTemplate(module, segment string, tmpl *template.Template, format, token string, claims, params interface{}) (interface{}, error) {
+func GoTemplate(ctx context.Context, tmpl *template.Template, format, token string, claims, params interface{}) (interface{}, error) {
```

Comment:
You took a context but are not using it. I suggest you use it instead of `context.Todo`